pkg/engine/armvariables.go

```diff
@@ -136,6 +136,7 @@ func getK8sMasterVars(cs *api.ContainerService) (map[string]interface{}, error)
 		"' AUTHENTICATION_METHOD=',variables('customCloudAuthenticationMethod')," +
 		"' IDENTITY_SYSTEM=',variables('customCloudIdentifySystem')," +
 		"' NETWORK_API_VERSION=',variables('apiVersionNetwork')" +
```

Comment:
we're missing a comma here near the end (I know this is horrible):

should be:

`"' NETWORK_API_VERSION=',variables('apiVersionNetwork')," +`

Reply:
```suggestion
		"' NETWORK_API_VERSION=',variables('apiVersionNetwork')," +
```

Reply:
e2e should raise this error as well. :)

Reply:
@gossion it would have. We didn't run e2e on the last commit because Jack caught the error before triggering the tests.